tools/deploy_wiremock.py

```diff
@@ -0,0 +1,33 @@
+import utils
+import deployment_options
+import pvc_size_utils
+
+
+log = utils.get_logger('deploy_wiremock')
+
+
+def main():
+    deploy_options = deployment_options.load_deployment_options()
+
+    log.info('Starting wiremock deployment')
+
+    utils.set_profile(deploy_options.target, deploy_options.profile)
+
+    deploy_wiremock(deploy_options)
+
+    log.info('Completed to wiremock deployment')
+
+
+def deploy_wiremock(deploy_options):
+    docs = utils.load_yaml_file_docs('deploy/wiremock/wiremock-deployment.yaml')
+
+    utils.set_namespace_in_yaml_docs(docs, deploy_options.namespace)
+
+    dst_file = utils.dump_yaml_file_docs('build/wiremock-deployment.yaml', docs)
```

Comment:
should this catch exceptions and fail in those cases? It does open files and more..

Reply:
hmm, haven't see any special handling in the other deploy scripts. but good point, maybe worth handling for all scripts.

Reply:
ack. this can happen in a different PR.